jsons/Units.json

```diff
@@ -2469,8 +2477,8 @@
 	},
 	{
 		"name": "Great Admiral", 
-		"unitType": "WaterCivilian",
-		"uniques": ["Bonus for units in 2 tile radius 15%","Heal adjacent units for an additional 15 HP per turn","Unbuildable"],
+		"unitType": "Civilian Water",
+		"uniques": ["Bonus for units in 2 tile radius 15%", "Heal adjacent units for an additional 15 HP per turn", "Unbuildable"],
```

Comment:
The "Heal adjacent units" unique is deprecated and should be replaced with `All adjacent units heal [+15] HP when healing`